MenuOOPs3/src/ShoppingStore/CartProducts.java

```diff
@@ -0,0 +1,42 @@
+package ShoppingStore;
+
+/**
+ * Extends the Product class and add more functionality - discount,quantity and
+ * totalAmount
+ * 
+ * @author Udit Saxena
+ *
+ */
+public class CartProducts extends Product {
```

Comment:
This class is used for keep data for single product, name should be CartProduct instead